src/app/content/components/Page/searchHighlightManager.spec.ts

```diff
@@ -74,8 +74,9 @@ describe('searchHighlightManager', () => {
     onHighlightSelect = jest.fn();
   });
 
-  it('calls highlight select callback only when a new highlight is selected + toggle solution if highlight is inside',
-    () => {
+  const testName = 'calls highlight select callback only when a new highlight is selected '
+    + '+ toggle solution if highlight is inside';
+  it(testName, () => {
```

Comment:
Tbh I'd prefer disabling eslint in this line, but this is just my personal opinion.

Reply:
eslint wasnt failing at this line

Reply:
But as I remember there was a newline between `'long-test-name'` and `()=>{`.